uber/templates/jobs/index.html

```diff
@@ -68,20 +68,20 @@
         </div>
 
 </div>
-{% comment %}
+{# comment
```

Comment:
Wait, is this commenting all this stuff out using Jinja2 and not just HTML? I can see that we had been doing this with Django, but do you know what's up here?